# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/hannes/embutils/makefile-head-install.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-08-14 16:36:38.000000000 +0200
+++ ./Makefile	2004-09-13 02:06:46.132798432 +0200
@@ -68,7 +68,7 @@
 clean:
 	rm -rf *.o bin-* $(PRGS)
 
-VERSION=embutils-$(shell head -1 CHANGES|sed 's/://')
+VERSION=embutils-$(shell head -n 1 CHANGES|sed 's/://')
 CURNAME=$(notdir $(shell pwd))
 
 rename:
@@ -89,6 +89,7 @@
 .PHONY: install
 
 install: $(TARGETS)
+	mkdir -p $(DESTDIR)$(prefix)/bin
 	$(INSTALL) $(TARGETS) $(DESTDIR)$(prefix)/bin
 
 uninstall:
